CONTRIBUTING.md

```diff
@@ -26,7 +26,7 @@ database. Use a text editor like emacs or NotePad instead._
 
 **Inclusion criteria**
 
-- [ ] Make sure that any faculty you add meet the inclusion
+- [X] Make sure that any faculty you add meet the inclusion
```

Comment:
This file needs to be removed from the PR.

Reply:
Deleted CONTRIBUTING.md. Thanks for processing this pull request.